### DISCUSSION TITLE
setConfig method

### DISCUSSION THREAD

**Discussion by LukeTOBrien** [GrapesJS/grapesjs#4722] [Ideas]
Hello,

You have a [getConfig method](https://grapesjs.com/docs/api/editor.html#getconfig) which returns an individual config property but what about a `setConfig(name, value)` method?  
I am just thinking about my project, I might want to set the `baseCss`.

Thanks,

Luke

**Comment by ronaldohoch** [comment]
Hello!
I think you can use this: https://grapesjs.com/docs/api/style_manager.html#addstyletargets
